templates/base/head_navbar.tmpl

```diff
@@ -121,7 +121,7 @@
 			<a href="{{AppSubUrl}}/notifications" class="item tooltip not-mobile gt-mx-0" data-content="{{.locale.Tr "notifications"}}" aria-label="{{.locale.Tr "notifications"}}">
 				<span class="fitted item">
 					{{svg "octicon-bell"}}
-					<span class="notification_count{{if not $notificationUnreadCount}} gt-hidden{{end}}">
+					<span class="notification_count{{if not $notificationUnreadCount}} gt-hidden{{end}}"><!-- Nested because the rendering breaks otherwise-->
```

Comment:
Some questions about this comment:

* What is `nested`?
* How rendering breaks?

At first glance, the layout should be like that (a bell before, the notification_count could be hidden), that's already clear.

Reply:
It is meant at the fact that we use a span inside a span.
I've checked if the nested span could be removed, which wasn't the case.

Reply:
That's pretty clear it must be done like this, I even couldn't think about another approach.